contracts/kernel/KernelStorage.sol

```diff
@@ -1,6 +1,19 @@
 pragma solidity 0.4.18;
 
 
-contract KernelStorage {
-    address public kernelImpl;
+contract KernelConstants {
+    bytes32 constant public CORE_NAMESPACE = keccak256("core");
+    bytes32 constant public APP_BASES_NAMESPACE = keccak256("base");
+    bytes32 constant public APP_ADDR_NAMESPACE = keccak256("app");
+
+    bytes32 constant public KERNEL_APP_ID = keccak256("kernel.aragonpm.eth");
+    bytes32 constant public KERNEL_APP = keccak256(CORE_NAMESPACE, KERNEL_APP_ID);
+
+    bytes32 constant public ACL_APP_ID = keccak256("acl.aragonpm.eth");
+    bytes32 constant public ACL_APP = keccak256(APP_ADDR_NAMESPACE, ACL_APP_ID);
```

Comment:
I'm a little confused by the naming, since `setApp()` takes in an ID, but we always use the non-ID version to actually locate something (e.g. we always use `setApp(KERNEL_APP)`).

Maybe `<name>_APP_NAME` and `<name>_APP_ID` would work better?